Resources/Audio/Effects/attributions.yml

```diff
@@ -216,3 +216,8 @@
   copyright: Bhijn and Myr (github username deathride58)
   license: CC0-1.0
   source: https://github.com/space-wizards/space-station-14/pull/23212
+
+- files: ["metal-pipe-falling.ogg"]
+  copyright: "Taken from thenotcheeseman via freesound.org and mixed from stereo to mono."
+  license: CC0-1.0
+  source: "https://freesound.org/people/thenotcheeseman/sounds/679206/"
```

Comment:
I'm reasonably sure this is not licensed properly and is a stolen sound effect from the original pack from the 90s.

Reply:
Unfortunately, I could not find another original. (＞︿＜)
Is this source definitely not suitable as the original?